src/coreclr/jit/regalloc.cpp

```diff
@@ -162,6 +162,18 @@ regNumber Compiler::raUpdateRegStateForArg(RegState* regState, LclVarDsc* argDsc
 #if FEATURE_MULTIREG_ARGS
     if (varTypeIsStruct(argDsc->lvType))
     {
+#ifdef TARGET_LOONGARCH64
+        {
+            if (argDsc->GetOtherArgReg() != REG_NA)
+            {
+                inArgMask = genRegMask(argDsc->GetOtherArgReg());
+                if (emitter::isFloatReg(argDsc->GetOtherArgReg()))
+                    codeGen->floatRegState.rsCalleeRegArgMaskLiveIn |= inArgMask;
+                else
+                    codeGen->intRegState.rsCalleeRegArgMaskLiveIn |= inArgMask;
+            }
+        }
+#else
```

Comment:
Instead of using this function, and adding an ifdef here, could you instead share  `LinearScan::unixAmd64UpdateRegStateForArg()`? It looks like it does the same thing you're doing.

Reply:
Should I add a new function liking `loongarch64UpdateRegStateForArg(LclVarDsc* argDsc)` or `LoongArch64UpdateRegStateForArg(LclVarDsc* argDsc)` ?

Reply:
Looks like you added `LoongArch64UpdateRegStateForArg`, but it is just a copy of `unixAmd64UpdateRegStateForArg`.

My proposal:
1. Delete LoongArch64UpdateRegStateForArg
2. Rename unixAmd64UpdateRegStateForArg to UpdateRegStateForStructArg and use it for both UNIX_AMD64_ABI and TARGET_LOONGARCH64.

Reply:
OK, Thanks

We don't modify other‘s architecture implementation until you ask.

Reply:
> We don't modify other‘s architecture implementation until you ask.

Thank you; that's a good, safe policy